parsing/dialects/intermediate/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ParserTest.java

```diff
@@ -114,6 +114,37 @@
     sql(sql).ok(expected);
   }
 
+  @Test public void testDeleteWithTable() {
+    final String sql = "delete foo from bar";
+    final String expected = "DELETE `FOO` FROM `BAR`";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testDeleteWithTableCompoundIdentifier() {
+    final String sql = "delete foo.bar from baz";
+    final String expected = "DELETE `FOO`.`BAR` FROM `BAZ`";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testDeleteWithTableWithAlias() {
+    final String sql = "delete foo from bar as b";
```

Comment:
Is it possible to have an alias for the first table as well?

Reply:
I checked the docs and it isn't.